privaterelay/templates/includes/messages.html

```diff
@@ -1,11 +1,13 @@
 {% if messages %}
 	{% for message in messages %}
 		<div class="messages home-messages">
-			<div{% if message.tags %} class="{{ message.tags }} js-notification" {% else %} class="" {% endif %}>
-				<span>{{ message }}</span>
-				<button class="flx center-value dismiss js-dismiss">
-					<svg class="x-close-icon fill-current text-gray hover:text-purple-800" role="button" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path d="M14.348 14.849a1.2 1.2 0 0 1-1.697 0L10 11.819l-2.651 3.029a1.2 1.2 0 1 1-1.697-1.697l2.758-3.15-2.759-3.152a1.2 1.2 0 1 1 1.697-1.697L10 8.183l2.651-3.031a1.2 1.2 0 1 1 1.697 1.697l-2.758 3.152 2.758 3.15a1.2 1.2 0 0 1 0 1.698z"/></svg>
-				</button>
+			<div{% if message.tags %} class="{{ message.tags }}" {% else %} class="" {% endif %}>
+				<div class="js-notification">
+					<span>{{ message }}</span>
+					<button class="flx center-value dismiss js-dismiss">
+						<svg class="x-close-icon fill-current text-gray hover:text-purple-800" role="button" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path d="M14.348 14.849a1.2 1.2 0 0 1-1.697 0L10 11.819l-2.651 3.029a1.2 1.2 0 1 1-1.697-1.697l2.758-3.15-2.759-3.152a1.2 1.2 0 1 1 1.697-1.697L10 8.183l2.651-3.031a1.2 1.2 0 1 1 1.697 1.697l-2.758 3.152 2.758 3.15a1.2 1.2 0 0 1 0 1.698z"/></svg>
+					</button>
+				</div>
```

Comment:
The `.js-notification` class has a `max-width` on it, so I had to put a div in between set the color. 

Alternate option would be to move the `{% if message.tags %} ` to the `<div class="messages home-messages">` div.